packages/paste-core/components/button/src/ResetButton.tsx

```diff
@@ -6,11 +6,15 @@ import {DirectButtonProps, DirectButtonPropTypes} from './types';
 // This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export
 const merge = require('lodash.merge');
 
-const defaultStyles: BoxStyleProps = merge({}, BaseStyles.default);
+const defaultStyles: BoxStyleProps = merge(BaseStyles.default, {
```

Comment:
You shouldn't remove the first `{},` because `lodash.merge` mutates the first object passed. BaseStyles.default will be affected for the other variants.  If this currently works it might just be a fluke that reset is parsed last.

Reply:
Pushed